### DISCUSSION TITLE
Conflicting notations in the doc: symmetric difference and Minkowski sum

### DISCUSSION THREAD

**Discussion by stla** [CGAL/cgal#7251] [General]
Both the symmetric difference and the Minkowski sum are denoted by $\oplus$ in the doc.

AFAIK, the most used notation for the symmetric difference is $\triangle$. And for the Minkowski sum, this is simply $+$.

**Comment by efifogel** [comment]
The Minkowsky sum notation is fine.
The symmetric difference notation should not be oplus.
   ____  _        ____             _
  /_____/_) o    /__________  __  //
 (____ (   (    (    (_/ (_/-(-'_(/
                         _/

**Comment by stla** [comment]
I find the $+$ notation is appropriate for Minkowski, because the Minkowski sum is nothing but a set of ordinary additions. Such a notation is also used in group theory. 

But well, "les goûts et les couleurs". The only important point is the conflict.

The oplus notation is sometimes used for the symmetric difference but I never encountered it (after a phd and 10 years of research in mathematics).

**Comment by efifogel** [comment]
The notation of "oplus" is common in publications about Minkowski sums;
see, e.g., "Handbook of discrete and computational geometry" page 1041.
   ____  _        ____             _
  /_____/_) o    /__________  __  //
 (____ (   (    (    (_/ (_/-(-'_(/
                         _/
